locales/en/LC_MESSAGES/http_validator.po

```diff
@@ -191,3 +191,12 @@ msgstr "###### Recommended CSP Rating:{0}"
 
 msgid "TEXT_REVIEW_CSP_RECOMMENDED_TEXT"
 msgstr "\r\n##### Want to improve your Content-Security-Policy game?\r\nWhy not try the below Content-Security-Policy response header to get started using Content Security Policy?\r\nRemember, below recommendation is only based on a sample (read: {0} page(s)) of all pages on your website\r\nand based without clicking on anything.\r\n\r\n###### Recommended Content-Security-Policy policies:\r\n{1}\r\n{2}\r\nRead more: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy\r\n\r\n"
+
+msgid "TEXT_REVIEW_SRI_COMPLIANT"
+msgstr "- {0}, Subresource Integrity (SRI)"
+
+msgid "TEXT_REVIEW_SRI_NONE_COMPLIANT"
+msgstr "- {0}, Needs Subresource Integrity (SRI)"
```

Comment:
@cockroacher @marcusosterberg can this be made more clear?

Reply:
after reviewing the code more, maybe it should say something about external resources are missing integrity attribute. I also think it would be helpful to give some guideline here, number of offending resources or what resources/tags are problematic.

Reply:
We now show SRI candidates and SRI error messages in "details" mode.
- webperf.se, Subresource Integrity (SRI) is required for following entries ( 1.00 rating )
  - `<link href="https://cdn.webperf.se/2.0/css/bootstrap.min.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/owl.carousel.min.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/owl.theme.default.min.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/style.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/widgets.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/color-default.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/responsive.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/css/google-fonts.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/fontello.css" rel="stylesheet">`
  - `<link href="https://cdn.webperf.se/2.0/css/custom.css" rel="stylesheet">`
  - `<script src="https://cdn.webperf.se/2.0/js/jquery-2.2.4.min.js">`
  - `<script src="https://cdn.webperf.se/2.0/js/bootstrap.js">`
  - `<script src="https://cdn.webperf.se/2.0/js/jquery-scrolltofixed-min.js">`
  - `<script src="https://cdn.webperf.se/2.0/js/theia-sticky-sidebar.js">`
  - `<script src="https://cdn.webperf.se/2.0/js/scripts.js">`